packages/jaeger-ui/src/components/TracePage/index.js

```diff
@@ -45,6 +45,7 @@ import type { Config } from '../../types/config';
 import prefixUrl from '../../utils/prefix-url';
 
 import './index.css';
+import getFilteredSpans from './TraceTimelineViewer/get-filtered-spans';
```

Comment:
Doesn't look like this file needs to be in the `./TraceTimelineViewer/` folder anymore?